cloudify_rest/tasks.py

```diff
@@ -93,7 +94,7 @@ def execute_as_relationship(params=None, template_file=None, save_path=None,
 
 
 def _execute(params, template_file, instance, node, save_path=None,
-             prerender=False, remove_calls=False):
+             prerender=False, remove_calls=False, auth=False):
```

Comment:
@0lvin Do you mean here to pass `auth=None` instead of `auth=False` ?

Reply:
Updated in 309bf0a